packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -180,6 +188,24 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
     setGetSceneObjectFunction(sceneLoader.getSceneObject);
   }, [sceneLoader]);
 
+  useEffect(() => {
```

Comment:
is there a reason to seperate these effects out instead of just a single effect on scene metadata module?

Reply:
Updated. Saved the `sceneMetadataModule` object instead of separate functions in GlobalSettings.